### DISCUSSION TITLE
after switching my site to https, all redirects return 404!

### DISCUSSION THREAD

**Discussion by outlander65** [YOURLS/YOURLS#2998] [Q&A] [answered]
Hi, after switching my site to https, all NEW redirects return 404! but old redirects are working with https.
can anyone help me with that? thanks in advance.

**Comment by outlander65** [comment, marked as answer]
I added
RewriteRule (.*) https://%{HTTP_HOST}%{REQUEST_URI}
just under
RewriteCond %{HTTPS} !on
in the .htaccess file and now redirects are working!

**Reply by ozh** [reply]
Note that this isn't standard .htaccess -- see https://github.com/YOURLS/YOURLS/wiki/.htaccess
